indico/testing/fixtures/user.py

```diff
@@ -60,3 +61,13 @@ def _create_group(id_):
 def dummy_group(create_group):
     """Create a mocked dummy group."""
     return create_group(1337)
+
+
+@pytest.fixture
+def create_identity(db):
+    """Return a callable which lets you create dummy identities."""
+    def _create_identity(user, provider, identifier):
+        identity = Identity(user=user, provider=provider, identifier=identifier)
```

Comment:
I would keep the `db.session.flush()` here, in tests it's useful that those things are immediately sent to the DB and not only implicitly later when they are needed..

Reply:
That makes sense, I added it back :+1: .